src/SelectMulti.vue

```diff
@@ -195,8 +199,12 @@
 				 * emits the most recently removed value,
 				 * *generally not necessary*, if state can be handled w/ v-model alone
 				 */
-				this.$emit('remove', this.selectedOptions[index])
+				const option = this.selectedOptions[index]
+				this.notificationMesasage = `option ${option.label} removed`
```

Comment:
Sweet! Great call. It definitely needs to indicate *somehow* that something happened. I'd like to do a quick call on this part.

Given that the default screen reader behavior for a lot of interactive elements seems to be:
1. Read out the content of the element when focused
2. When an action is taken, just say the action, rather than the action + the content again

I think I *might* vote for this to just be `option removed` or just `removed`

but would like to discuss on a call